_notifications-plugin/index.md

```diff
@@ -0,0 +1,135 @@
+
+1. In the **Availability** section, select the OpenSearch plugins you want to use with the notification channel.
+2. Choose **Create**.
+
+### Amazon SNS as a channel type
+
+OpenSearch supports Amazon SNS for notifications. This integration with Amazon SNS means that, in addition to the other channel types, the notifications plugin can send emails, text messages, and even run AWS Lambda functions using SNS topics. For more information about Amazon SNS, see the [Amazon Simple Notification Service Developer Guide](https://docs.aws.amazon.com/sns/latest/dg/welcome.html).
+
+The notifications plugin currently supports two ways of user authentication:
+
+1. Provide the user with full access to Amazon SNS.
+2. Let the user assume an IAM role that has permissions to access Amazon SNS. Once you configure the notification channel to use the right Amazon SNS permissions, select the OpenSearch plugins that can trigger notifications.
+
+### Provide full Amazon SNS access permissions
+
+If you want to provide full Amazon SNS access to the IAM user, ensure that the user has the following permissions.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "sns:*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+### Assuming an IAM role with Amazon SNS permissions
+
+If you want to let the user send notifications without directly having full permissions to Amazon SNS, let the user assume a role that does have the necessary permissions.
+
+The IAM user must have the following permissions to assume a role.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:Describe*",
+        "iam:ListRoles",
+        "sts:AssumeRole"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+Then add this policy into the IAM user’s trust relationship to actually assume the role.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+  {
+    "Effect": "Allow",
+    "Principal": {
+    "AWS": "arn:aws:iam::<arn_number>:user/<iam_username>",
+    },
+    "Action": "sts:AssumeRole"
+  }
+  ]
+}
+```
+
+
+## Email as a channel type
+
+To send or receive notifications with emails, choose **Email** as the channel type. Next, select at least one sender and default recipient. To send notifications to more than a few people at a time, select a recipient group. If the Notifications plugin doesn’t currently have the necessary senders or groups, you can add them by first selecting **SMTP sender**, then choose **Create SMTP sender** or **Create recipient group**. Choose **SES sender** to use Amazon Simple Email Service (SES).
+
+### Create email sender
+
+1. Specify a unique name to associate with the sender.
+2. Enter an email address, and, if applicable, its host (for example, smtp.gmail.com), and the port. If you're using SES, enter the IAM role ARN of the AWS account to send notifications from, along with the region.
```

Comment:
Remove the first comma. Remove the comma after the parenthetical. Change "SES" to "Amazon SES". If this is its first appearance on the page, define "ARN" as "Amazon Resource Name (ARN)". If "region" refers to an AWS Region, please change to "AWS Region."